react_practice/react-apps/waitlist-page-v1/src/CTAForm.js

```diff
@@ -45,11 +40,10 @@ function CTAForm() {
           value={value}
           onChange={handleChange}
         />
-        <PrimaryButton
+        <PrimaryBtn
           title="Join the waitlist"
           className="cta-form-btn"
-          onKeyPress={handleKeyPress}
-          handleSubmit={handleSubmit}
+          onClick={handleSubmit}
```

Comment:
As a best practice, I generally like to name the event handlers as `on<Action>` where `Action` is something descriptive. Since this is meant to model the click behavior on the button, this would be named `onClick`. It reads nicely when you're looking at the component as well.

As you've done here, I also find it good practice to handle the handler `handle<Action>` like you've done here. Nice job!

Reply:
yea i realize now after pushing it up that i had left this this way. This was actually me trying to pass the handle submit as a prop to the child PrimaryBtn component so that it would fire the same response whether 'enter' was pressed or the user clicked on the submit button. i actually removed that after pushing it up but was so focused on trying to get it to work i forgot to tell you.